src/scss/layout.scss

```diff
@@ -65,7 +65,7 @@ button {
   cursor: pointer;
 }
 
-.empty-item {
+.empty-item, .permission-denied-item {
```

Comment:
Ich hijacke diese Zeile mal für eine andere.

In Zeile 126 ist ein typo (`inclue`) und funzt deshalb vll. nicht, weiß aber garnicht, ob das überhaupt verwendet wird.

Reply:
hab ich noch mitgefixt